website/docs/rbi.md

```diff
@@ -126,14 +127,17 @@ The most common way to get RBI files for gems is to generate (or regenerate)
 them. To regenerate RBI files in an already-initialized project:
 
 ```bash
-# Update only the autogenerated gem RBIs:
-srb rbi gems
+# Update only the autogenerated gem RBIs for new gems:
+❯ bin/tapioca gems
```

Comment:
What is `bin/tapioca`? Should this be `bundle exec tapioca`?

Reply:
When you run `tapioca init`, Tapioca will create a [binstub](https://bundler.io/man/bundle-binstubs.1.html) for itself so you can run it through `bin/tapioca`

Reply:
Great thanks, do you mind mentioning that `tapioca init` will create that in the `adopting.md` page?

Reply:
Done in 4cf15c320 👍